src/neuroconv/tools/nwb_helpers/_configuration_models/_hdf5_dataset_io.py

```diff
@@ -75,6 +75,9 @@ def get_data_io_kwargs(self) -> Dict[str, Any]:
             elif self.compression_method is None:
                 compression_bundle = dict(compression=False)
         else:
-            compression_bundle = dict(compression=self.compression_method, compression_opts=self.compression_options)
+            compression_opts = None
```

Comment:
```suggestion
            # Base filters only take particular form of a single input; single int for GZIP; 2-tuple for SZIP
            compression_opts = None
```

Reply:
Does this help?

Reply:
Well, technically the 78-81 lines are covered by the `test_minimal` run, it's just that the coverage report is only generated once by the full test run (unless you want the bot to spam twice from two uploads per push)

I guess I should I add a note and `pragma` for that... but rest assured that block is tested

As for 66-76 I think I ought to be able to add a test harnessing the [PyNWB tutorial for dynamic filters](https://pynwb.readthedocs.io/en/stable/tutorials/advanced_io/h5dataio.html#dynamically-loaded-filters), I guess I just didn't think of that. Might take a little bit